src/services/users.service.ts

```diff
@@ -1,11 +1,19 @@
-import { BaseUserDto } from '@/dtos';
-import { UserEntity } from '@/entities';
+import { BaseUserDto, BasePostDto } from '@/dtos';
+import { UserEntity, PostEntity } from '@/entities';
 
 class UserService {
   public async getUsersFromDB(): Promise<BaseUserDto[]> {
     const users: BaseUserDto[] = await UserEntity.find();
     return users;
   }
+  public async getPostsByUser(userId: number): Promise<BasePostDto[]> {
+    const posts: BasePostDto[] = await PostEntity.find({
```

Comment:
Instead of using find for specific things, you can use findOne, then pass in conditions. 
This keeps all querey requests for getting something specific the same

Reply:
Nvm, this route is getting based off a user id and not a post id. MB, Your good!